scripts/stubtest/allowlist.txt

```diff
@@ -268,3 +268,9 @@ django.contrib.gis.db.backends.base.features.BaseSpatialFeatures.unsupported_geo
 # Django uses a `*args` parameter but crash if it contains more than 1 element.
 django.http.QueryDict.pop
 django.http.request.QueryDict.pop
+
+# Django incorrectly decorates this with `@classmethod`.
+django.test.selenium.SeleniumTestCase.__init_subclass__
```

Comment:
`__init_subclass__` shouldn't be decorated with `@classmethod` even though it is one - it works implicitly. This is something that should be fixed in Django, but perhaps mypy is also unnecessarily picky here?

Reply:
yes, this is a `stubtest` bug. Can you please fix it upstread? I will be happy to review and merge your changes.